modules/enabling-multi-node-pytorch-training-jobs-with-rdma.adoc

```diff
@@ -0,0 +1,237 @@
+RDMA improves the training job performance because it eliminates the overhead of using the operating system CPUs and memory.
+Running a training job on multiple nodes using multiple GPUs can significantly reduce the completion time.
+
+When submitting a PyTorch training job to a cluster configured for RDMA, ensure that the job is configured to use the high-speed network interfaces, as described in this section. 
+
+
+.Prerequisites
+
+* You can access an OpenShift cluster that has multiple worker nodes with supported NVIDIA GPUs.
+
+* Your cluster administrator has configured the cluster as follows:
+
+ifdef::upstream[]
+** Installed {productname-long} with the required distributed training components, as described in link:{odhdocshome}/installing-open-data-hub/#installing-the-distributed-workloads-components_install[Installing the distributed workloads components].
+endif::[]
+ifdef::self-managed[]
+** Installed {productname-long} with the required distributed training components, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-the-distributed-workloads-components_install[Installing the distributed workloads components] (for disconnected environments, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/installing-the-distributed-workloads-components_install[Installing the distributed workloads components]).
+endif::[]
+ifdef::cloud-service[]
+** Installed {productname-long} with the required distributed training components, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-the-distributed-workloads-components_install[Installing the distributed workloads components].
+endif::[]
+
+ifdef::upstream[]
+** Configured the distributed training resources, as described in link:{odhdocshome}/managing-odh/#managing_distributed_workloads[Managing distributed workloads].
+endif::[]
+ifndef::upstream[]
+** Configured the distributed training resources, as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-distributed-workloads_managing-rhoai[Managing distributed workloads].
+endif::[]
+
+ifdef::upstream[]
+** Configured the cluster for RDMA, as described in link:{odhdocshome}/managing-odh/#configuring-a-cluster-for-rdma_managing-odh[Configuring a cluster for RDMA].
+endif::[]
+ifndef::upstream[]
+** Configured the cluster for RDMA, as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-distributed-workloads_managing-rhoai#configuring-a-cluster-for-rdma_managing-rhoai[Configuring a cluster for RDMA].
+endif::[]
+
+.Procedure
+. Log in to the OpenShift Console.
+
+. Find the PyTorch job that you want to edit, as follows:
+.. In the *Administrator* perspective, click *Workloads -> Jobs*.
```

Comment:
PyTorchJobs won't be found here. They will be found by going to home -> search -> Resources -> PyTorchJob. 
I am also unsure as to why we are editing an existing PyTorchJob rather than adding a new one to the cluster?

Reply:
Updated, thank you!